sample_data/Readme.md

```diff
@@ -0,0 +1,7 @@
+This folder contains sample data sets which import with the import command fromt the server repository, e.g.: `utils/inv-import.js file.csv YYYY-MM-DD`
+
+### Medications
+
+source | filename | descritpion | header information | notes 
+-------|----------|-------------|--------------------|-------
+http://www.pbs.gov.au/browse/downloads | amt_20170201.csv | Australian PBS database | see [here](https://dev.pbs.gov.au/readme.html) | this database includes all Australian PBS drugs.  The delimiter is `!`. To use the file it needs to be slightly modified (commas replaced with semi-colons). 
```

Comment:
Looking briefly at the data, it appears that the column header names in the CSV files need to modified as well in order for the import tool to understand the data being imported in.